packages/smooth_app/lib/pages/preferences/user_preferences_prices.dart

```diff
@@ -171,6 +171,11 @@ class UserPreferencesPrices extends AbstractUserPreferences {
         ),
         Icons.open_in_new,
       ),
+      _getListTile(
+          'General Data Protection Regulation',
+          () async => LaunchUrlHelper.launchURL(
+              'https://wiki.openfoodfacts.org/GDPR_request'),
+          Icons.open_in_new),
```

Comment:
A good pactice is to stick to the style of the rest of the project.
```suggestion
          Icons.open_in_new,),
```

Reply:
okay i will follow the same code style and stick to it thanks